metaflow/plugins/kfp/kfp_decorator.py

```diff
@@ -65,9 +66,6 @@ def __init__(self, attributes=None, statically_defined=False):
         super(KfpInternalDecorator, self).__init__(attributes, statically_defined)
 
     def step_init(self, flow, graph, step, decos, environment, datastore, logger):
-        if datastore.TYPE != "s3":
```

Comment:
Did we want to remove this check? Seems like it would still be useful if a customer is trying to run a flow locally but still includes the `kfp run` command.

Reply:
Could you elaborate more - how would the user use kfp run locally? I assumed that user will always have `--s3-code-package` since it's a default option, and `--s3-code-package` does check for `datastore.TYPE == s3` here https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/plugins/kfp/kfp_cli.py#L296-L299

Reply:
I'm removing this check just so that the `--yaml-only` option doesn't require s3 when `--s3-code-package` is turned off - a case where I just want to inspect the yaml spec for compiled kfp pipeline.